src/components/Prompt/index.jsx

```diff
@@ -29,9 +29,9 @@ class Prompt extends React.Component {
     e.preventDefault();
     const { emptyInputAlertMessage, onClickCloseButton, onClickConfirmButton, dispatchShowToast } = this.props;
     const { promptInput: inputValue } = this.state;
```

Comment:
아니면 아예 책장 이름을 좌우 공백을 제거한 것으로...

```suggestion
    const inputValue = this.state.promptInput.trim();
```

Reply:
한번 백엔드 정책 확인 해보고 맞추는게 좋을 것 같네요!